src/store/index.js

```diff
@@ -8,6 +8,7 @@ import Transaction from 'ethereumjs-tx'
 import abiDecoder from 'abi-decoder'
 import util from 'ethereumjs-util'
 import Bluebird from 'bluebird'
+import _ from 'lodash'
```

Comment:
Probably it has no sense because `lodash` is required by `aepp-identity` dependences:
```
aepp-identity$ npm ls lodash --prod
aepp-identity@0.0.1 /home/denis/projects/aeternity/aepp-identity
├─┬ abi-decoder@1.0.9
│ ├─┬ babel-core@6.26.0
│ │ ├─┬ babel-generator@6.26.0
│ │ │ └── lodash@4.17.4  deduped
│ │ ├─┬ babel-register@6.26.0
│ │ │ └── lodash@4.17.4  deduped
│ │ ├─┬ babel-template@6.26.0
│ │ │ └── lodash@4.17.4  deduped
│ │ ├─┬ babel-traverse@6.26.0
│ │ │ └── lodash@4.17.4  deduped
│ │ ├─┬ babel-types@6.26.0
│ │ │ └── lodash@4.17.4  deduped
│ │ └── lodash@4.17.4  deduped
│ └─┬ babel-preset-es2015@6.24.1
│   ├─┬ babel-plugin-transform-es2015-block-scoping@6.26.0
│   │ └── lodash@4.17.4  deduped
│   ├─┬ babel-plugin-transform-es2015-classes@6.24.1
│   │ └─┬ babel-helper-define-map@6.26.0
│   │   └── lodash@4.17.4  deduped
│   └─┬ babel-plugin-transform-es2015-sticky-regex@6.24.1
│     └─┬ babel-helper-regex@6.26.0
│       └── lodash@4.17.4  deduped
├─┬ eth-lightwallet@3.0.1
│ └─┬ bitcore-lib@0.15.0
│   └── lodash@4.17.4  deduped
├── lodash@4.17.4 
└─┬ web3-provider-engine@13.8.0
  └─┬ async@2.6.0
    └── lodash@4.17.4  deduped
```

for tests I have created a new [vuejs-templates/webpack](https://github.com/vuejs-templates/webpack) project and checked imports:
```
import _ from 'lodash';
import { times } from 'lodash';
```
and corresponding bundle size using `npm run build --report`. In both cases `lodash` is imported fully until I import it by particular path:
```
import times from 'lodash/times';
```
so this seems to be the most efficient way to import lodash functions, except [babel-plugin-lodash](https://github.com/lodash/babel-plugin-lodash) because it transforms all kinds of imports to import by a particular file. It significantly reduces bundle size in the empty project but doesn't work in `aepp-identity` as I think because of other packages dependencies.
Also, I tried `lodash-es` package but for some reason, its behavior is the same as `lodash`.

Reply:
I propose to merge this as is and open a new issue about `lodash` shaking.